src/js/structs/Pod.js

```diff
@@ -138,6 +138,10 @@ module.exports = class Pod extends Service {
     return this._spec;
   }
 
+  getStatusString() {
+    return (this.get('status') || '').toUpperCase();
```

Comment:
I don't really think it makes sense to create a utility function for this. The logic is pretty simple (just upper casing and checking for default value).

/cc @orlandohohmeier - what do you think?